test/integration/scheduler_perf/scheduler_test.go

```diff
@@ -167,25 +161,13 @@ func baseConfig() *testConfig {
 	}
 }
 
-func defaultSchedulerBenchmarkConfig(numNodes, numPods int) *testConfig {
-	baseConfig := baseConfig()
-
-	nodePreparer := framework.NewIntegrationTestNodePreparer(
-		baseConfig.schedulerSupportFunctions.GetClient(),
-		[]testutils.CountToStrategy{{Count: numNodes, Strategy: &testutils.TrivialNodePrepareStrategy{}}},
-		"scheduler-perf-",
-	)
-
-	config := testutils.NewTestPodCreatorConfig()
-	config.AddStrategy("sched-test", numPods, testutils.NewSimpleWithControllerCreatePodStrategy("rc1"))
-	podCreator := testutils.NewTestPodCreator(baseConfig.schedulerSupportFunctions.GetClient(), config)
-
-	baseConfig.nodePreparer = nodePreparer
-	baseConfig.podCreator = podCreator
-	baseConfig.numPods = numPods
-	baseConfig.numNodes = numNodes
-
-	return baseConfig
+// getBaseConfig returns baseConfig after initializing number of nodes and pods.
+// We have to function for backward compatibility. We can combine this into baseConfig
+func getBaseConfig(nodes int, pods int) *testConfig {
```

Comment:
As updated in the comments, this will be removed, I am having them only for backward compatibility.